darts/models/forecasting/ensemble_model.py

```diff
@@ -197,5 +197,22 @@ def ensemble(
     def min_train_series_length(self) -> int:
         return max(model.min_train_series_length for model in self.models)
 
+    @property
+    def extreme_lags(self):
+        lag_aggregators = (min, max, min, min, max)
+        return [
+            self._find_max_lag_or_none(i, agg) for i, agg in enumerate(lag_aggregators)
+        ]
+
+    def _find_max_lag_or_none(self, lag_id, aggregator):
```

Comment:
Could you add a unit test for this? It would be great to check that ensemble models now get the correct extreme lags (for example for the regression model example I mentioned in a previous comment?) No need to call backtest/historical_forecasts, just a simple check that we get the correct lags.